syntax = "proto2";

package BaseNetSceneReq;


message BaseNetSceneReq {
  optional int32 net_scene_type = 1;
  optional bytes net_scene_req_buff = 2;

}
